### PR TITLE
SwitchContinueToBreakFixer - do not support alternative syntax

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5076

### CODE REVIEW

**Comment by kubawerlos** [comment]
Shouldn't the fixer actually support alternative syntax in case the `NoAlternativeSyntaxFixer` is not used?

**Comment by SpacePossum** [comment]
Ideally it would support the alternative syntax, however the time I can spend on this PR is very limited. Maybe someone who actually uses the alternative syntax can send a PR later to add the support for it :)

File: tests/Fixer/ControlStructure/SwitchContinueToBreakFixerTest.php [hunk 1]
```diff
@@ -35,6 +35,27 @@ public function testFix($expected, $input = null)
     public function provideTestFixCases()
     {
         return [
+            'alternative syntax |' => [
+                '<?php
+                    switch($foo):
+                        case 3:
+                            continue;
+                    endswitch?>
+                '
```
Comment (GrahamCampbell) [inline tests/Fixer/ControlStructure/SwitchContinueToBreakFixerTest.php:44]: Missing comma.

Comment (SpacePossum) [reply tests/Fixer/ControlStructure/SwitchContinueToBreakFixerTest.php]: thanks, fixed!

**Comment by julienfalque** [comment]
Thank you @SpacePossum.

**Pull request merged**